src/DynamoCoreWpf/Views/Core/NoteView.xaml.cs

```diff
@@ -15,6 +17,7 @@ namespace Dynamo.Nodes
 {
     public partial class NoteView : IViewModelView<NoteViewModel>
     {
+        private const double MINIMUM_ZOOM_DIRECT_NODE_EDIT = 0.5;
```

Comment:
Would you put some comments for this? Thanks!

Reply:
ok